### ISSUE TITLE
Column('Change (1,5, etc)m, %') always returns an error when using .get_scanner_data()

### ISSUE THREAD

**Issue by Frank-Raddi** [shner-elmo/TradingView-Screener#20]
I was testing with the scanner and can't seem to get get passed this error. It seems to be occurring for all the 'Change xm, %'s.

Error:
`Exception has occurred: HTTPError
400 Client Error: Bad Request
 Body: {"totalCount":0,"error":"Unknown field \"change.5\"","data":null}
 for url: https://scanner.tradingview.com/america/scan
  File "N:\StockTrading\Live\StockScreener\StockScreener.py", line 20, in <module>
    .get_scanner_data())
requests.exceptions.HTTPError: 400 Client Error: Bad Request
 Body: {"totalCount":0,"error":"Unknown field \"change.5\"","data":null}
 for url: https://scanner.tradingview.com/america/scan`

Full code: 
```
from tradingview_screener import Scanner, Query, Column

_, scalpers = (Query()
 .select('name')
 .where(
     Column('Relative Volume') > 1.5,
     Column('relative_volume_10d_calc') > 1.5,
     Column('Change %') >= 8,
 )
 .get_scanner_data())

breakouts = (Query()
 .select('name')
 .where(
     Column('Relative Volume') > 3,
     Column('relative_volume_10d_calc') > 3,
     Column('Relative Volume at Time') > 3,
     Column('Change 5m, %') >= 5
 )
 .get_scanner_data())
#
#_, breakouts = breakouts  # Unpack the tuple to ignore the count and get the DataFrame
## Extract the 'name' column
#tickers = tickers.extend(breakouts['name'].tolist())

print(scalpers['name'].tolist())
```

**Comment by shner-elmo** [comment]
I fixed it. (A lot of the fields have been renamed from `name.timeframe` to `name|timeframe`)

**Comment by Frank-Raddi** [comment]
Awesome. Great work!

**Comment by shner-elmo** [comment]
Thanks. I released version 2.4.1 which has the updated columns, good luck.
